hydroflow/src/scheduled/port.rs

```diff
@@ -2,31 +2,43 @@
 use std::marker::PhantomData;
 
 use ref_cast::RefCast;
+use sealed::sealed;
 
 use crate::scheduled::handoff::{CanReceive, Handoff, TryCanReceive};
 
 use super::HandoffId;
 
+#[sealed]
+pub trait Polarity: 'static {}
+
+pub enum SEND {}
```

Comment:
do these need to be caps? don't feel strongly really but I haven't seen types capsed before.

Reply:
They don't need to but its a convention Ive been using for types which are actually used as tag names (rather than being used like normal structs and instantiated). Its also in https://github.com/hydro-project/hydroflow/blob/main/hydroflow/src/lang/tag.rs